config/settings.yml

```diff
@@ -1355,9 +1355,10 @@ rrd:
       - fake_email
 
 virtual_regional_office:
-  url: http://va.gov
-  token: 9d3868d1-ec15-4889-8002-2bff1b50ba62
+  url: https://sandbox.lighthouse.va.gov/abd-vro
+  api_key: 9d3868d1-ec15-4889-8002-2bff1b50ba62
```

Comment:
Is this a fake API key?

Reply:
@LindseySaari 

It's a local-development-only API key. It's hard-coded here to match [this value in the VRO server code](https://github.com/department-of-veterans-affairs/abd-vro/pull/171/files#diff-57d9a737e396d6733d202bd587fc4ae91d4ea909a90e5db05488cd8682cd2f6eR2). The intention is to allow devs on this project to run the VRO server locally side-by-side with vets-api.

For higher environments, we'll do the usual settings templating with proper secrets-management.